### PR TITLE
Sandbox pseudo device copy fix

### LINKED ISSUE #2031: 3.0: copy.Copy() causing issues with sandbox build
## Version of Singularity:
v3.0.0-alpha.1-550-g73e3ad9

### Expected behavior
A sandbox image is built... 

```
$ sudo singularity build --sandbox nginx nginx.def
  [ ... ]
INFO:    Creating sandbox directory...
INFO:    Build complete!
```

### Actual behavior
Build hangs, and continuously copies `/dev/full` into the sandbox

```
$ ls -lh nginx/dev/full 
-rw-rw-rw- 1 root root 115G Sep 19 21:22 nginx/dev/full
```

### Steps to reproduce behavior
Using below Def file, build a sandbox image cross-device, so the `copy.Copy()` call is triggered.

```
---- Start Cut ----
BootStrap: debootstrap
OSVersion: stable
MirrorURL: http://ftp.us.debian.org/debian/

%runscript
    echo "This is what happens when you run the container..."

%post
    echo "Hello from inside the container"
    apt-get update
    apt-get -y install vim procps netcat iptables net-tools iputils-ping nginx systemd systemd-sysv
    apt-get clean
    systemctl enable nginx
---- End Cut ----
```

### PR DESCRIPTION
**Pull request by ikaneshiro**
This PR uses the 'mv' command instead of a combination of `rename()` and `copy()` in order to allow for building sandboxes across devices and to prevent the issue of attempting to read from pseudo devices such as `/dev/full` while moving the sandbox cross device.

This also removes the `github.com/otiai10/copy` dependency.

Fixes #2031

### CODE REVIEW

**Review by ArangoGutierrez — COMMENTED** [review]
I am getting this ERROR while trying to build the definition file example on #2031 
```bash
Fetched 5396 kB in 3s (1469 kB/s)         
Reading package lists... Done
E: Unable to change to (unreachable)/tmp/ - chdir (2: No such file or directory)
FATAL:   post proc: exit status 100
FATAL:   While performing build: while running engine: exit status 255
```

**Comment by jmstover** [comment]
@ArangoGutierrez See #2176 ... and #2158 for the PR with the fix for that issue.

File: src/pkg/build/assemblers/assembler_sandbox.go [hunk 1]
```diff
@@ -28,16 +29,9 @@ func (a *SandboxAssembler) Assemble(b *types.Bundle, path string) (err error) {
...
-			return err
-		}
-
-		if err := os.RemoveAll(b.Rootfs()); err != nil {
-			sylog.Errorf("Unable to remove Bundle directory: %s", err)
-			return err
-		}
+	cmd := exec.Command("mv", b.Rootfs(), path)
+	if err = cmd.Run(); err != nil {
+		return fmt.Errorf("Sandbox Assemble Failed: %s", err)
```
Comment (jmstover) [inline src/pkg/build/assemblers/assembler_sandbox.go:34]: Use `fmt` ... or `sylog` that you're already importing?

Comment (bauerm97) [reply src/pkg/build/assemblers/assembler_sandbox.go:34]: Here is **returning** `fmt.Errorf()`, meaning we're using the format package to return an `error` type which contains formatted text. We can't return `sylog.Errorf()`, which is a function which prints output and does not return anything.

Comment (jmstover) [reply src/pkg/build/assemblers/assembler_sandbox.go:34]: :+1:

**Review by jmstover — APPROVED** [review]
LGTM

**Review by bauerm97 — APPROVED** [review]

**Pull request merged**